salt/modules/artifactory.py

```diff
@@ -38,7 +38,7 @@ def __virtual__():
         return True
 
 
-def get_latest_snapshot(artifactory_url, repository, group_id, artifact_id, packaging, target_dir='/tmp', target_file=None, classifier=None, username=None, password=None):
+def get_latest_snapshot(artifactory_url, repository, group_id, artifact_id, packaging, target_dir='/tmp', target_file=None, classifier=None, username=None, password=None, use_literal_group_id=False):
```

Comment:
Is this expected to work on Windows? If so, I'm wondering if `target_dir` should be `None` and then we transform it in the implementation? If not, we should specifically exclude Windows in `__virtual__`.

I know that's not part of your change but I just saw it. :]